index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+    <div class="Header">
+        <h1  class="mylogo" >MY logo</h1>
+        <div>
+            <img src="./image/logobuger.JPG">
+        </div>
+    </div>
```

Comment:
:red_circle:  Please make sure you implement all the requested changes. The previous reviewer requested that you
 - [ ] use the header tag instead of the div tag for semantic reasons. 
 - [ ]  use the `a` tag for your logo instead h1. So that someone can easily navigate your page by clicking on your logo to get back to the homepage.
 - [ ] use the nav tag to wrap your hamburger for semantic reasons.

Reply:
:red_circle: Please make sure that you work on these requirements as they are part of the project specifications.

You can, for example, use this to update your code. 

   ```
 <header class="Header">
        <a  class="mylogo" >MY logo</a>
        <nav>
            <img src="./image/logobuger.JPG">
        </nav>
    </div>

```